### PR TITLE
Integrate phasePlannerStore for Enhanced Ticket Management and Websocket Support

### PR DESCRIPTION
**Pull request by aliraza556**
### Describe the Chnages:
- Integrated phasePlannerStore for centralized ticket management and real-time updates via websockets in the Phase Planner.

closes: #700

## Issue ticket number and link:
- **Ticket Number:** [ 700]
- **Link:** [ https://github.com/stakwork/sphinx-tribes-frontend/issues/700]

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome
- [x] New feature (non-breaking change which adds functionality)
- [x] I have provided a screenshot of changes in my PR

### CODE REVIEW

**Comment by aliraza556** [comment]
Hi @humansinstitute, Please review this PR.

**Comment by aliraza556** [comment]
Hi @humansinstitute, Please review this PR.

**Comment by aliraza556** [comment]
@humansinstitute Locally, Prettier passed, but I’m not sure why it failed in the CI/CD pipeline.

image

File: src/config/socket.ts
Comment (humansinstitute) [inline src/config/socket.ts:1]: We're not looking to change the SOCKET_MSG in this change. 

The plan is to keep this message and associated code in place. Out of scope change.

File: src/people/widgetViews/PhasePlannerView.tsx [hunk 1]
```diff
@@ -63,6 +64,17 @@ const PhasePlannerView: React.FC = () => {
         const sessionId = res.body;
         console.log(`Websocket Session ID: ${sessionId}`);
       }
+
+      if (res.msg === SOCKET_MSG.ticket_update && res.body) {
```
Comment (humansinstitute) [inline src/people/widgetViews/PhasePlannerView.tsx:68]: This is not the approach we will be taking please remove. Out of Scope.

**Pull request merged**